lispy-test.el

```diff
@@ -1481,7 +1481,21 @@ Insert KEY if there's no command."
   (should (string= (lispy-with "(list\n foo\n |1 (2\n    3)\n bar)" ";")
                    "(list\n foo\n |;; 1 (2\n ;;    3)\n bar)"))
   (should (string= (lispy-with "(defun foo ()|\n  (bar))" ";")
-                   "(defun foo ()\n  ;; |\n  (bar))")))
+                   "(defun foo ()\n  ;; |\n  (bar))"))
+  ;; With single semicolons
+  (setq lispy-comment-use-single-semicolon t)
```

Comment:
This should be a `let` instead of `setq`, so that the setting applies only to these statements, and not to all the following tests.